src/pages/profile/help/index.tsx

```diff
@@ -1,22 +1,83 @@
 import React from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import { IHelpDataProps } from '@allTypes/reduxTypes/supportStateTypes'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { dispatch } from '@redux/hooks'
+import { supportMiddleware } from '@redux/slices/support'
 import { Button } from '@uiComponents/Button'
 import { Input } from '@uiComponents/Input'
 
-export const Help = () => (
-  <div className="grid gap-4 rounded bg-gray-thin p-5 xl:p-10">
-    <h2 className="text-lg">Tell us about your problem:</h2>
-    <p className="text-base text-black-light">
-      Try to support your explanation with a screenshot, with that we can solve your problem faster.
-    </p>
-    <Input placeholder="Headline" />
-    <Input
-      placeholder="Type your problem here..."
-      color="primary"
-      rows={7}
-    />
-    <Button>Submit</Button>
-  </div>
-)
+import { helpValidationSchema } from '../../../validation/support/help'
```

Comment:
Add validation to tsconfig to have
```suggestion
import { helpValidationSchema } from '@validation/support/help'
```

Reply:
done